spec/services/contact_details/update_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "rails_helper"
+
+module ContactDetails
+  describe Update do
+    describe ".call" do
+      let(:trainee) { create(:trainee) }
+
+      context "when international address attribute provided but with uk locale code" do
+        let(:attributes) { { locale_code: "uk", international_address: "some international address" } }
+
+        before do
+          described_class.call(trainee: trainee, attributes: attributes)
+        end
+
+        it "does not save the international address value" do
+          expect(trainee.reload.international_address).to be_nil
+        end
+      end
+
+      context "when uk address values provided but with non uk locale code" do
+        let(:attributes) { { locale_code: "non_uk", international_address: "some international address" } }
```

Comment:
Ummm where is the uk address?

:thinking:

Reply:
The factory defaults to UK values so I assume that here